dm/dm/master/openapi_view.go

```diff
@@ -60,6 +61,34 @@ func (s *Server) redirectRequestToLeaderMW() gin.HandlerFunc {
 	}
 }
 
+// reverseRequestToLeaderMW reverses request to leader.
+func (s *Server) reverseRequestToLeaderMW() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		ctx2 := c.Request.Context()
+		isLeader, _ := s.isLeaderAndNeedForward(ctx2)
+		if isLeader {
+			c.Next()
+		} else {
+			// nolint:dogsled
+			_, _, leaderOpenAPIAddr, err := s.election.LeaderInfo(ctx2)
+			if err != nil {
+				_ = c.AbortWithError(http.StatusBadRequest, err)
+				return
+			}
+			// simpleProxy just reverses to leader host
+			simpleProxy := httputil.ReverseProxy{
+				Director: func(req *http.Request) {
+					req.URL.Scheme = "http"
```

Comment:
we should take care of the `https` because user may config tls connection between master